### PR TITLE
docs: add vision-camera-plugin-builder sections in Creating Frame Processor Plugins

### PR DESCRIPTION
**Pull request by mateusz1913**
## What

👋 I recently made [a small CLI](https://github.com/mateusz1913/vision-camera-plugin-builder) in order to make creation of frame processor plugins quicker. I thought it would be great to add this to docs. Let me know wdyt

## Changes

This PR adds automatic setup section to "Creating Frame Processor Plugins (iOS|Android)" with
vision-camera-plugin-builder CLI

| iOS | Android |
| --- | -------- |
| Screen Shot 2022-07-06 at 20 10 39 | Screen Shot 2022-07-06 at 20 11 41 |

## Tested on

Run yarn --cwd docs start

### CODE REVIEW

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx [hunk 1]
```diff
@@ -14,6 +14,34 @@ In this guide we will create a custom QR Code Scanner Plugin which can be used f
...
+### Mostly automatic setup
+
+1. Run [Vision Camera Plugin Builder CLI](https://github.com/mateusz1913/vision-camera-plugin-builder)
+
+```sh
+npx vision-camera-plugin-builder android
+```
+
+:::info
+CLI will ask for params like path to project's Android Manifest file, name of the plugin (e.g. `QRCodeFrameProcessor`), name of the exposed method (e.g. `scanQRCodes`) & language you want to use for plugin development (you can choose Java or Kotlin).
```
Comment (mrousavy) [inline docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx]:
```suggestion
The CLI will ask you for the path to project's Android Manifest file, name of the plugin (e.g. `QRCodeFrameProcessor`), name of the exposed method (e.g. `scanQRCodes`) and language you want to use for plugin development (Java or Kotlin).
```

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx [hunk 2]
```diff
@@ -14,6 +14,34 @@ In this guide we will create a custom QR Code Scanner Plugin which can be used f
...
+For params reference check [CLI's docs](https://github.com/mateusz1913/vision-camera-plugin-builder#%EF%B8%8F-options).
```
Comment (mrousavy) [inline docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_ANDROID.mdx]:
```suggestion
For reference see the [CLI's docs](https://github.com/mateusz1913/vision-camera-plugin-builder#%EF%B8%8F-options).
```

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx [hunk 3]
```diff
@@ -14,6 +14,20 @@ In this guide we will create a custom QR Code Scanner Plugin which can be used f
...
+### Automatic setup
+
+Run [Vision Camera Plugin Builder CLI](https://github.com/mateusz1913/vision-camera-plugin-builder), 
+
+```sh
+npx vision-camera-plugin-builder ios
+```
+
+:::info
+CLI will ask for params like path to project's .xcodeproj file, name of the plugin (e.g. `QRCodeFrameProcessor`), name of the exposed method (e.g. `scanQRCodes`) & language you want to use for plugin development (you can choose Objective-C, Objective-C++ or Swift).
```
Comment (mrousavy) [inline docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx]:
```suggestion
The CLI will ask you for the path to project's .xcodeproj file, name of the plugin (e.g. `QRCodeFrameProcessor`), name of the exposed method (e.g. `scanQRCodes`) and language you want to use for plugin development (Objective-C, Objective-C++ or Swift).
```

File: docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx [hunk 4]
```diff
@@ -14,6 +14,20 @@ In this guide we will create a custom QR Code Scanner Plugin which can be used f
...
+For params reference check [CLI's docs](https://github.com/mateusz1913/vision-camera-plugin-builder#%EF%B8%8F-options).
```
Comment (mrousavy) [inline docs/docs/guides/FRAME_PROCESSOR_CREATE_PLUGIN_IOS.mdx]:
```suggestion
For reference see the [CLI's docs](https://github.com/mateusz1913/vision-camera-plugin-builder#%EF%B8%8F-options).
```

**Review by mrousavy — CHANGES_REQUESTED** [review]
Hi - thanks for your PR! The CLI looks really awesome, I'm so happy creating FP Plugins just got way easier!! 🔥 

Left a few comments for the phrasing, otherwise it looks good!

**Comment by mateusz1913** [comment]
Fixed up commit with suggested changes

**Pull request merged**